presto-main/src/main/java/com/facebook/presto/dispatcher/QueryAndSessionProperties.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.dispatcher;
+
+import com.facebook.presto.common.analyzer.PreparedQuery;
+
+import java.util.Map;
+import java.util.Optional;
+
+public class QueryAndSessionProperties
```

Comment:
This is done differently, the reason is we have SessionBuilder at dispatchManager, which can directly set the required properties.